tuf-spec.md

```diff
@@ -490,9 +490,11 @@ A delegated role file is located at:
 # Document formats # {#document-formats}
 
 All of the formats described below include the ability to add more
-attribute-value fields for backwards-compatible format changes.  If
-a backwards incompatible format change is needed, a new filename can
-be used.
+attribute-value fields to dictionary objects for backwards-compatible format
+changes.  Implementers who encounter undefined key-value pairs in the format
+must include the data when calculating hashes or verifying signatures and must
+preserve the data when re-serializing. If a backwards incompatible format change
```

Comment:
I'm not sure what this last sentence means exactly, but we'll leave it for another PR, since you didn't write it. Maybe someone should make an issue to try to clarify it, though.

Reply:
This looks like a sentence left from pre-semantic versioning. I opened #170.